### ISSUE TITLE
include and exclude no longer accepts string patterns

### ISSUE THREAD

**Issue by raix** [TypeStrong/fork-ts-checker-webpack-plugin#548]
Not sure if it's a breaking change or if something happened in validation.

previously `reportFiles` could take in glob patterns (not sure if it's micromatch)

From the docs it's hard to read what can be in an array (for include/exclude)

But converting to 6.0.5 the following gives validation errors:
```js
issue: {
  include: [
    '**/src/**/*.{ts,tsx}',
  ],
```

I've updated to the following but not sure if it's the correct approach (it seems like micromatch is supported for file)

```js
issue: {
  include: [
    '**/src/**/*.{ts,tsx}',
  ].map(file => ({ file })),
```

The above also applies for `exclude`
